### ISSUE TITLE
Doesn't build on case insensitive file systems

### ISSUE THREAD

**Issue by sternenseemann** [kamwitsta/fuzzytime#2]
macOS / darwin has a case insensitive file system, so it can't deal with `src/fuzzytime.hs` and `src/FuzzyTime.hs` at the same time. You probably should rename the file implementing the `Main` module to fix this.

**Comment by kamil-stachowski** [comment]
I've upload a new version. I will not have access to a MacOS/Darwin machine for another two weeks, can you please check that the problem has been solved?

**Comment by sternenseemann** [comment]
Yes, latest master builds on darwin. Note that the (API breaking) rename of `FuzzyTime` to `LibFuzzyTime` is not necessary.

**Comment by kamil-stachowski** [comment]
I'm afraid I don't know how else to do this. If you have a better solution, and would be so kind as to it implement it, it would be great.

**Comment by sternenseemann** [comment]
You just need to rename `fuzzytime.hs` to `main.hs` (like you've done), the `FuzzyTime` directory could have been left as is, I think.

**Comment by kamil-stachowski** [comment]
You're right, I don't know why I thought I had to rename `FuzzyTime`. I think it should be ok now, thanks.
